<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QDialog" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="TitleBar" name="titleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="logInB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="warnL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码错误</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="capsLockHintL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>大写锁定已打开</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="AnimationRefresh" name="refreshAR">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>247</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <zorder>refreshAR</zorder>
  <zorder>titleBar</zorder>
  <zorder>backgroundLabel</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>logInB</zorder>
  <zorder>warnL</zorder>
  <zorder>capsLockHintL</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QGroupBox</extends>
   <header location="global">ui/titlebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnimationRefresh</class>
   <extends>QGroupBox</extends>
   <header location="global">ui/animationrefresh.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
